Parser/src/main/java/org/openzen/zenscript/parser/expression/ParsedExpressionVariable.java

```diff
@@ -40,7 +40,7 @@ public CompilingExpression compile(ExpressionCompiler compiler) {
 
 	@Override
 	public CompilingExpression compileKey(ExpressionCompiler compiler) {
-		if (!typeArguments.isEmpty())
+		if (typeArguments != null && !typeArguments.isEmpty())
```

Comment:
💭 Question:
Is there a semantic difference between `null` and empty type arguments?
Currently, the parser parses `<>` as empty and null if no `<` is encountered

Reply:
There is no semantic difference